test/ccapi/unittest_ccapi.cpp

```diff
@@ -205,8 +205,8 @@ TEST(nntrainer_ccapi, train_with_config_01_p) {
   EXPECT_EQ(model->initialize(), ML_ERROR_NONE);
   EXPECT_NO_THROW(model->train());
 
-  EXPECT_NEAR(model->getTrainingLoss(), 4.434051, tolerance);
-  EXPECT_NEAR(model->getValidationLoss(), 2.910938, tolerance);
+  EXPECT_NEAR(model->getTrainingLoss(), 4.1389656, tolerance);
```

Comment:
i wonder why the accuracies changed.

Reply:
It's been a while but afai remember, it was because the shuffling has changed from handcrafted to std::shuffle.

Reply:
Can you once turn off shuffle in the old and new code and verify that the values match for the tests where you modified the values. This is critical for correct working.

Reply:
Confirming it's exactly correct with shuffling is off.

leaving a record to reproduce

TC name: nntrainer_capi_nnmodel.train_with_file_01_p

1. 2.3630399703979492 (train loss)
2. 2.2508399486541748 (val loss)
3. 8.3332996368408203 (val acc)